lib/shoryuken/manager.rb

```diff
@@ -78,6 +78,7 @@ def processor_done(queue, processor)
     def processor_died(processor, reason)
       watchdog("Manager#processor_died died") do
         logger.error { "Process died, reason: #{reason}" unless reason.to_s.empty? }
+        reason.backtrace.each { |bt| logger.error { bt } } unless reason.backtrace.nil?
```

Comment:
@phstc sorry, been horribly busy and keep forgetting about this.  It is as you suspected before but I'd still wager it's better to guard against this and output a backtrace whenever you can.  It's certainly helped me in the last ~5 months, but if you're dead set against it, I'll defer to your judgement.